facebook-instant-articles.php

```diff
@@ -386,6 +388,9 @@ function inject_ia_markup_meta_tag() {
 	}
 	add_action( 'wp_head', 'inject_ia_markup_meta_tag' );
 
+	// Injects the <link rel...> tag if the AMP Markup is enabled
+	add_action( 'wp_head', array('Instant_Articles_Amp_Markup', 'inject_link_rel') );
```

Comment:
We could encapsulate all the action binding within the Instant_Article_Amp_Markup class, so the functionality is self-contained. We can create a static `init()` method to handle all the binding.